uw-research-computing/os-transition-htc.md

```diff
@@ -0,0 +1,145 @@
+also include a notification to the `chtc-users` mailing list. 
+
+## Important Dates
+
+* **August 2022**: significant HTC capacity running CentOS Stream 8, available for testing 
+and general use.
+* **September 15, 2022**: Default operating system requirements for jobs change from CentOS 7 to CentOS Stream 8
+* **September 2022**: More than 75% of CHTC capacity is running on CentOS Stream 8
+
+## What You Need to Do
+
+### Actions For All Users
+
+All users of CHTC's HTC system should **test their existing workflows**. 
+The only exception are those users who use Docker or Singularity containers 
+to run their jobs. 
+
+1. Set up a set of typical jobs to use as a test set. It is ideal 
+to use a set of jobs for which you already have results so that you confirm 
+the results of the test jobs. 
+
+2. Modify the job submit file(s) to require servers that are running the 
+soon-to-be default operating system, CentOS Stream 8. 
+[See below for details](#require-centos-stream-8-upcoming-default))
+
+3. Submit the jobs. Monitor them for issues -- check both the results of 
+your code and the standard output and error files for any unexpected behavior. 
+
+If you have any concerns or questions from your job tests, please contact 
+the facilitation team at chtc@cs.wisc.edu. 
+
+If your tests go well, you should be well poised for the transition and can either 
+wait for the default setting for jobs to change, or opt into using both in 
+the meantime. See immediately below for how to do this. 
+
+### Get More Computing (Optional)
+
+Additionally, for those who want to maximize their usage of HTC system 
+resources, we recommend **opting into use of computers with both operating systems**. 
+
+Only do this after you have first run a set of test jobs on CentOS Stream 8 specifically. 
+Once that has been done, see below for how to run on servers running both 
+versions of Linux: [Use Both CentOS 7 (current default) and CentOS Stream 8 (upcoming 
+default)](#use-both-centos-7-current-default-and-centos-stream-8-upcoming-default))
+
+If you use Docker or Singularity containers in your jobs, you do **not** need to 
+specify use of both operating systems. These jobs will run on servers using either 
+operating system without any changes to your submit file. 
+
+## Current Status of the HTC System
+
+### Capacity Available in the HTC System
+
+<table class="gtable">
+  <tr>
+    <th>Linux Version</th>
+    <th>Percent of Pool Capacity</th>
+    <th>Notable Servers</th>
+  </tr>
+  <tr>
+    <td>CentOS 7</td> 
+    <td>75%</td>
+    <td>Build nodes, range of GPU servers, high memory nodes</td>
+  </tr>
+  <tr>
+    <td>CentOS Stream 8</td> 
+    <td>25%</td>
+    <td>Build node, Servers with A100 GPUs</td>
+  </tr>
+</table>
+
+### Default Operating System
+
+Currently, CHTC-managed submit servers automatically add a job 
+requirement requesting servers with our most recent operating system,
+CentOS 7. To override this default, see below: [Requesting a Specific
```

Comment:
```suggestion
By default, CHTC-managed submit servers automatically add a job 
requirement that requires jobs to run on servers running our primary operating system,
CentOS 7, unless otherwise specified by the user. To override this default, see below: [Requesting a Specific
```